### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by gogasca**

### CODE REVIEW

**Comment by gogasca** [comment]
@sirtorry 
@thedriftofwords 
Feel free to send over some recommendations after our Friday meeting. Thanks

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -30,35 +30,66 @@ consistency and quality for the notebook assets:
...
-   
+ 
    If Colab is required please provide compatibility with AI Platform
    notebooks, to help you on this we have created a template for you!
-   [Template here]((notebooks/templates/ai_platform_notebooks_template_hybrid.ipynb)
+   [Template here](notebooks/templates/ai_platform_notebooks_template_hybrid.ipynb)
+   
+   ## Notebook requirements
+   1. Use [AI Platform notebooks](https://cloud.google.com/ai-platform-notebooks/)
+   We highly recommend to guarantee that your Notebook runs in AI Platform notebooks
```
Comment (sirtorry) [inline CONTRIBUTING.md]: notebook runs in AI Platform Notebooks

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -30,35 +30,66 @@ consistency and quality for the notebook assets:
...
-   3. Documentation
-   4. Asset is executable without errors 
-   5. Use latest ML framework
-   6. Follow Software engineering principles
-      - Python 3.X
+   2. Use [templates](notebooks/templates) listed above. 
+   3. Provide novel and relevant content 
+   4. Provide Notebook documentation
+   5. Use latest ML framework version
+   6. Follow Software engineering principles:
```
Comment (sirtorry) [inline CONTRIBUTING.md]: software

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -30,35 +30,66 @@ consistency and quality for the notebook assets:
...
+   - AutoML Tables
+   - BigQuery
+   - BigQuery ML
+   
+   For the following products, select the ML framework and place it accordingly:
+   
+   - AI Platform Training
+   - AI Platform Prediction
+   - AI Platform Explanations
+   - AI Platform notebooks
```
Comment (sirtorry) [inline CONTRIBUTING.md]: AI Platform Notebooks

**Pull request merged**